modules/common/src/main/PackageObject.scala

```diff
@@ -188,6 +188,9 @@ trait WithPlay { self: PackageObject =>
     def toHundredths: Long = self.toMillis / 10
   }
 
+  implicit def LilaFiniteDurationZero: Zero[FiniteDuration] =
+    Zero.instance(FiniteDuration(0, scala.concurrent.duration.MILLISECONDS))
+
```

Comment:
Way to go. Since it's a constant, it could be a `val` instead of `def` as to be constructed only once.